src/Files.Uwp/Helpers/FileExtensionHelpers.cs

```diff
@@ -66,7 +66,7 @@ public static bool IsBrowsableZipFile(string filePath, out string ext)
             }
 
             ext = new[] { ".zip", ".7z", ".rar", ".tar" } // Only ext we want to browse
-                .FirstOrDefault(x => filePath.Contains(x, StringComparison.OrdinalIgnoreCase));
+                .FirstOrDefault(x => filePath?.Contains(x, StringComparison.OrdinalIgnoreCase) ?? false);
```

Comment:
Good catch

Reply:
We ended up fixing this issue twice, on main and here. No need for ` filePath?` since above we check  `if (string.IsNullOrEmpty(filePath))`.